### ISSUE TITLE
Array method is broken

### ISSUE THREAD

**Issue by ghostdevv** [ghoststools/dashargs#5]
![image](https://user-images.githubusercontent.com/47755378/99299266-e5020700-2842-11eb-8d8b-110b462a7bd1.png)

For the given test you should expect a output of
```js
DashArgs { b: true } [ { key: 'b', value: true, raw: '-b' } ]
DashArgs { c: true } [ { key: 'c', value: true, raw: '-c' } ]
DashArgs { b: true } [ { key: 'b', value: true, raw: '-b' } ]
```

but you get 

```js
DashArgs { b: true } [ { key: 'b', value: true, raw: '-b' } ]
DashArgs { c: true } [ { key: 'c', value: true, raw: '-c' } ]
DashArgs { b: true } [ { key: 'c', value: true, raw: '-c' } ]
```

This is because the DashArgs class uses a private storage method of an object that is defined outside of the class, therefore is globally accesible.

**Comment by ghostdevv** [comment]
Will be fixed in 4.0.0

**Comment by ghostdevv** [comment]
Fixed https://github.com/ghoststools/dashargs/commit/b5538122a3267a8ec053b91e4afdccae97b664a5
